app/assets/javascripts/divvy-app.coffee

```diff
@@ -15,6 +15,12 @@ divvyApp
         longitude: -87.648056
       zoom: 14
 
+    # serverError is null by default
+    $scope.serverError = null
+
+    $scope.dismissError = ->
```

Comment:
I think the Angular way to do this would be to have a directive for the modal, like you suggested, and `$watch` a `modalVisible` property. I wouldn't call this in its current state _deplorable_, although any avid Yeoman user may. :trollface: